tutorials/11-probabilistic-pca/11_probabilistic-pca.jmd

```diff
@@ -353,43 +325,42 @@ You can find more details about this in e.g. [5].
 end;
 ```
 
-Instead of drawing samples of each entry in $w$ from a standard normal, this time we repeatedly draw $D$ samples from the D-dimensional MND, forming a $D \times D$ matrix $w$.
-This matrix is a function of $\alpha$ as the samples are drawn from the MND parameterized by $\alpha$.
+Instead of drawing samples of each entry in $W$ from a standard normal, this time we repeatedly draw $D$ samples from the $D$-dimensional MND, forming a $D \times D$ matrix $W$.
```

Comment:
```suggestion
Instead of drawing samples of each entry in $\mathbf{W}$ from a standard normal, this time we repeatedly draw $D$ samples from the $D$-dimensional MND, forming a $D \times D$ matrix $\mathbf{W}$.
```